src/neuroglancer/ui/segment_split_merge_tools.ts

```diff
@@ -58,9 +58,9 @@ export class MergeSegmentsTool extends Tool<SegmentationUserLayer> {
       if (!Uint64.equal(segmentSelectionState.selectedSegment, mappedAnchorSegment)) return;
       const base = segmentSelectionState.baseSelectedSegment;
       const isBase = isBaseSegmentId(base);
-      const highBitVal = segmentEquivalences.disjointSets.highBitRepresentative.value;
```

Comment:
This logic isn't quite right, since currently the bitvector only talks about representative vs non-representative, not high bit.  But in practice this will still work.  Can you just add a TODO, like:

// TODO: This would ideally rely on a separate `HIGH_BIT_REPRESENTATIVE` flag, but it nonetheless still works correctly for nggraph and local equivalences.